server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,36 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.post("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    const parkId = req.params.parkId;
+    await Review.query().deleteById(reviewId);
+    const park = await Park.query().findById(parkId);
+    const serializedPark = await ParkSerializer.showDetails(park);
+    return res.status(201).json({ park: serializedPark });
+  } catch (error) {
+    return res.status(500).json({ errors: error });
+  }
+});
+
+parkReviewsRouter.patch("/:reviewId", async (req, res) => {
+  try {
+    const parkId = req.params.parkId;
+    const reviewId = req.params.reviewId;
+    const comments = req.body.comments;
+    const rating = req.body.rating;
+    await Review.query().patch({ comments: comments }).findById(reviewId);
+    await Review.query().patch({ rating: rating }).findById(reviewId);
+    const park = await Park.query().findById(parkId);
+    const serializedPark = await ParkSerializer.showDetails(park);
+    return res.status(201).json({ park: serializedPark });
+  } catch (error) {
```

Comment:
we shouldnt need to pass back the whole park, just the updated review, and maybe a calculated average review. NO PARK